### PR TITLE
Fix random spaces in golem code

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Fix random spaces in golem code

## Why It's Good For The Game

it annoyed me

### CODE REVIEW

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 1]
```diff
@@ -56,51 +56,53 @@
 	w_class = WEIGHT_CLASS_BULKY
 
 /obj/item/golem_shell/attackby(obj/item/I, mob/user, params)
-	..()
+	. = ..()
```
Comment (LemonInTheDark) [inline code/modules/ruins/lavaland_ruin_code.dm:59]: Won't this end up changing behavior?

**Comment by LemonInTheDark** [comment]
Conflict du marge

File: code/modules/ruins/lavaland_ruin_code.dm (hunk 1 above)
Comment (TiviPlus) [reply code/modules/ruins/lavaland_ruin_code.dm:59]: No

Comment (LemonInTheDark) [reply code/modules/ruins/lavaland_ruin_code.dm:59]: if you say so

**Pull request merged**